### DISCUSSION TITLE
Can't list or uninstall global packages

### DISCUSSION THREAD

**Discussion by FlanderJam** [coreybutler/nvm-windows#592] [Support]
Hello!

I'm running into some issues with NVM4W and am hoping to get some help.

Build Info:

- OS: Windows 10 Pro
- OS Version: 20H2
- OS Build: 19042.630
- NVM Version: 1.1.7

When I run `nvm install latest`
- It successfully installs node v15.3.0 (at time of writing, though the problem was also encountered against v15.2.1)
- It successfully installs npm v7.0.14

When I run `nvm use 15.3.0`
- It appears to successfully establish the symlink
- node commands appear to work
 - npm commands appear to work

However, if I try to install a package globally: `npm install -g typescript` for example
- The package appears to be installed and can be viewed in the nvm node home and in the nodejs symlink folder
- Running `npm list -g` returns empty. It's as if it doesn't recognize typescript is installed.
- Running `npm uninstall -g typescript` also fails to remove the global module. 
- This is happening for any package installed globally
- Running `tsc --version` will still return the newly installed version number for typescript

Should npm be able to install and uninstall packages globally? Should these packages appear when running commands like `npm list -g`?

**Comment by coreybutler** [comment]
This is likely caused by an upstream issue. See #579.
